tests/certification/state/azure/cosmosdb/cosmosdb_test.go

```diff
@@ -133,14 +133,17 @@ func TestAzureCosmosDBStorage(t *testing.T) {
 				"partitionKey": "mypartition",
 			}
 
-			test := func(setMeta, getMeta map[string]string, expectedValue string) {
+			test := func(setMeta, getMeta map[string]string, expectedValue string, expectedErr bool) {
```

Comment:
Why these changes?

Reply:
this was copy/paste from me looking at other prs to bring in changes for func signatures and copying this too. It's not needed so reverted.